content/blog/2025-07-31-valkey-swift/index.md

```diff
@@ -0,0 +1,185 @@
++++
+title= "Introducing valkey-swift, the Swift client for Valkey"
+description = "The new Swift client for Valkey is now in public preview. Learn about its features and how it brings the performance of Valkey to the Swift language."
+date= 2025-08-04 12:00:00
+authors= [ "adamfowler"]
+
+[extra]
+featured = false
++++
+We are excited to introduce the public preview of [valkey-swift](https://github.com/valkey-io/valkey-swift), a new Swift based client library for Valkey.
```

Comment:
Could you define what a "public preview" is in this context? It may help to clarify which features are implemented (as well as which are not).

Reply:
Also, I'd like 'Swift' to Swift.org